rust/lance/src/index/vector/ivf/v2.rs

```diff
@@ -799,9 +799,9 @@ mod tests {
     }
 
     #[rstest]
-    #[case(4, DistanceType::L2, 0.9)]
-    #[case(4, DistanceType::Cosine, 0.9)]
-    #[case(4, DistanceType::Dot, 0.8)]
+    #[case(4, DistanceType::L2, 0.7)]
+    #[case(4, DistanceType::Cosine, 0.7)]
+    #[case(4, DistanceType::Dot, 0.7)]
```

Comment:
let's low the recall requirement for only 4bit IVF_PQ:
- L2 -> 0.85
- Cosine -> 0.85
- Dot -> 0.75

Reply:
Ok, I've scoped the change to just 4bit IVF_PQ